README.md

```diff
@@ -10,8 +10,8 @@ from Vim into the system clipboard.
 Usage
 -----
 
-System copy provides a mapping to copy to the system clipboard. It accepts a
-motion or can be used with a visual selection.
+System copy provides a mapping to copy and paste to the system clipboard.
+It accepts a motion or can be used with a visual selection.
```

Comment:
I think this is mixing the two functions and might confuse since pasting doesn't accept a motion. Maybe update to:

``` md
System copy provides a mapping to copy to the system clipboard 
using a motion or visual selection. It also provides a mapping for
pasting from the system clipboard.
```